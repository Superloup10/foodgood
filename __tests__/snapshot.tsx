import "@testing-library/jest-dom";
import {render} from "@testing-library/react";
import Home from "@/app/[lang]/page";

it("renders homepage unchanged", () => {
    const {container} = render(<Home/>);
    expect(container).toMatchSnapshot();
});
