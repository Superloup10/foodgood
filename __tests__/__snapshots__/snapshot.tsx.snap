// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main>
    <h1>
      Foodgood
    </h1>
  </main>
</div>
`;
